express/features/sign-in.feature

```diff
@@ -0,0 +1,161 @@
+Feature: Happy paths for the self-service sign-in flow
+
+  Background:
+    Given that the user is on the "/sign-in" page
+
+  Scenario: User does not enter a valid email address
+    When the user submits the email "invalid-email.com"
+    Then the error message "Enter an email address in the correct format, like name@example.com" must be displayed for the "emailAddress" field
+
+  Scenario: The user does not enter any characters into the free text field
+    When the user submits the email ""
+    Then the error message "Enter your email address" must be displayed for the "emailAddress" field
+
+  Scenario: The user’s email address is saved with the service
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Continue button
+    Then they should be redirected to the "/sign-in-password" page
+
+  Scenario: User doesn’t enter any characters
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Continue button
+    Then they should be redirected to the "/sign-in-password" page
+    When they enter "" into the "password" field
+    When they click the Continue button
+    Then the error message "Enter your password" must be displayed for the "password" field
+
+  Scenario: User enters an invalid password
+    When they enter "password-will-be-wrong@stub.gov.uk" into the "emailAddress" field
+    When they click the Submit button
+    Then they should see the text "Enter your password"
+    When they enter "Invalid-Password" into the "password" field
+    When they click the Submit button
+    Then the error message "Incorrect password" must be displayed for the "password" field
+
+  Scenario: User enters a valid password
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Submit button
+    Then they should see the text "Enter your password"
+    When they enter "ValidPassword" into the "password" field
+    When they click the Submit button
+    Then they should be redirected to the "/sign-in-otp-mobile" page
+
+  Scenario: The user wants to see their password as they type it
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Submit button
+    Then they should see the text "Enter your password"
+    When they toggle on the "Show" link
+    When they enter "PasswordIsShown" into the "password" field
+    Then they see the toggle "Hide" link
+
+  Scenario: The user does not enter any characters into the free text field
```

Comment:
```suggestion
  Scenario: The user does not enter any characters when changing phone number
```

This test belongs in the "your account" feature rather than sign-in

Reply:
Moved to "your account".